src/pug/blog/report_22.pug

```diff
@@ -0,0 +1,49 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #22
+
+block content
+  h2 Progress report #22
+  +news_header('12 April 2024', 'report_21', '')
+
+  h3 Wireless Alpakka (the odyssey continues)
+  p In the last report we explained our criteria to evaluate #[em Pico-w + BT] (with and without dongle) and how we were not able to achieve acceptable results.
+
+  p During the last weeks we focused on evaluating #[em Pico-w + WLAN], and while the initial results on latency (~2 ms) and jitter were very satisfactory, we found critical issues with #[+ax('https://github.com/georgerobotics/cyw43-driver/issues/111') connection dropping] that we were no able to solve either, which also makes the solution unsuitable.
+
+  p Ultimately we have to ask ourselves if BT or WLAN are the #[em right tool] for low-latency videogame peripherals, and the answer is possibly "no".
+
+  p So even though we may go back to one of these solutions if the blockers are solved, for now we decided to explore alternatives without the Pico-W:
+
+  ul
+    li Keep the original Pico (or bare #[em RP2040]), but with a #[em standalone RF module] best suited for our usecase.
+    li Redesign the PCB into the #[em Nordic Semi] SoC ecosystem.
+    li Redesign the PCB into the #[em Espressif ESP32] SoC ecosystem.
+
+
+  p In the case of Nordic or ESP32 redesigns, we would also have to retain retrocompatibility for current PCBs, which is not a trivial task, so we are exploring how it could be done with #[+ax('https://www.zephyrproject.org') Zephyr] since it provides multiple architectures as a target.
```

Comment:
"We would endeavor to maintain a level of SW backward-compatibility to current PCBs, so that future bug-fixes and potentially also feature would continue to be applicable."
It's too early to go into details in a progress report, so drop the rest (Zephyr, core module, etc.). Otherwise this becomes too much conjecture of possible future work and too little actual progress report.